platform/ui/src/components/StudyListPagination/StudyListPagination.js

```diff
@@ -20,7 +20,7 @@ const StudyListPagination = ({
           <div className="flex items-center">
             <div className="relative mr-3">
               <select
-                defaultValue={perPage}
+                value={perPage}
```

Comment:
Just switches from default to controlled?

Reply:
yes